### PR TITLE
Removes the singularity beacon kit from the traitor bundle list.

### PR DESCRIPTION
**Pull request by Supermichael777**

### CODE REVIEW

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 /obj/item/weapon/storage/box/syndicate/New()
 	..()
-	switch (pickweight(list("bloodyspai" = 3, "stealth" = 2, "bond" = 2, "screwed" = 2, "sabotage" = 3, "guns" = 2, "murder" = 2, "implant" = 1, "hacker" = 3, "lordsingulo" = 1, "darklord" = 1, "sniper" = 1, "metaops" = 1, "ninja" = 1)))
+	switch (pickweight(list("bloodyspai" = 3, "stealth" = 2, "bond" = 2, "screwed" = 2, "sabotage" = 3, "guns" = 2, "murder" = 2, "implant" = 1, "hacker" = 3, /*"lordsingulo" = 1,*/ "darklord" = 1, "sniper" = 1, "metaops" = 1, "ninja" = 1)))
```
Comment (lzimann) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Remove don't comment it out.

**Comment by Repukan** [comment]
>Removes dark Lord as well

Fuck off with these stealth changes

**Comment by lzimann** [comment]
He's not removing dark lord.

**Comment by Repukan** [comment]
I'm a fool in a man's shoes

**Comment by coiax** [comment]
@Supermichael777 the new SM release kit is an RCD and a revolver. Just pry open the doors and shoot the SM multiple times until it explodes.

**Comment by Supermichael777** [comment]
with a camo kit you can do it through the doors

**Comment by coiax** [comment]
But the camo kit doesn't have damaging projectiles... right?

**Comment by jughu** [comment]
Rename murderbone kit into SM release kit 
Btw dont remov it

**Comment by Cyberboss** [comment]
Hold on a second, Cerestation has a classic engine. Why do this?

**Comment by PKPenguin321** [comment]
Cerestation isn't even in the game and it's WIP, and on every other map this is just a dud bundle that wastes your TCs, essentially

**Comment by scoopscoop** [comment]
Why not reduce it's TC costs and slap a big fat warning on it's description?

**Pull request merged**